chore(css): Move CSS examples - Basic shapes, filter effects, module template, paint API (#36770)
mdn/content: files/en-us/web/css/basic-shape/ellipse/index.md

### Basic ellipse() example

> This example shows an ellipse with an x radius of 40%, a y radius of 50% and the position being left. This means that the center of the ellipse is on the left edge of the box giving us a half ellipse shape to wrap our text around.

Reviewer: also, not part of this PR, but should we write:

```suggestion
This example shows an ellipse that is floated left that has a horizontal radius of 40%, a vertical radius of 50%, and a left position. This means that the center of the ellipse is on the left edge of the box giving us a half ellipse shape to wrap our text around.
```
